src/main/java/org/jenkinsci/plugins/testrail/JunitResults/Testcase.java

```diff
@@ -27,20 +27,24 @@
 public class Testcase {
     private String name;
     private Failure failure;
+    private Skipped skipped;
```

Comment:
Sure. The Skipped class is just there for marshalling. The meaningful parts are the element is looked for with [`@XmlElement(name = "skipped")`](https://github.com/jenkinsci/testrail-plugin/pull/20/files#diff-40b4e251736a3048cde778b8e8da0b5eR38) and that there's [a null check](https://github.com/jenkinsci/testrail-plugin/pull/20/files#diff-9fe7050b076aaf5d5693ff2c65860e26R207) to determine if that child `<skipped/>` element is present. There may be a more elegant way to do this but I followed the `Failure` method for simplicity.